lib/src/screens/dashboard/widgets/balance_page.dart

```diff
@@ -28,47 +31,58 @@ class BalancePage extends StatelessWidget {
           !dashboardViewModel.balanceViewModel.isReversing,
       child: SingleChildScrollView(
         child: Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.center,
           children: [
-            SizedBox(height: 56),
-            Container(
+            Observer(
+            builder: (_) => dashboardViewModel.type != WalletType.monero
```

Comment:
**Enhancement:**
for scalability, we can put this into the view model under a variable called `hasAccounts` 
this way we maintain separation of concerns as the logic would be in the view model
and we would maintain scalability so that when we get to add new wallet that has accounts feature we can just add that to the variable in the view model.

and then we can make just a simple if condition for a centered `HomeScreenAccountWidget`

something like this variable `isHomeScreenSettingsEnabled` in line `61`

Reply:
fixed